### PR TITLE
Adding tracing 

### PR DESCRIPTION
**Pull request by gklijs**
By setting a span factory on both the `DistributedComm…andBus` and the `JGroupsConnector`.
Fixes https://github.com/AxonFramework/extension-jgroups/issues/188

### CODE REVIEW

File: jgroups/src/main/java/org/axonframework/extensions/jgroups/commandhandling/JGroupsConnector.java [hunk 1]
```diff
@@ -331,23 +338,42 @@ private void processReplyMessage(JGroupsReplyMessage message) {
...
+            commandMessage = message.getCommandMessage(serializer);
+        } catch (Exception e) {
+            if (message.isExpectReply()) {
+                sendReply(msg.getSrc(), message.getCommandIdentifier(), asCommandResultMessage(e));
+            } else {
+                logger.error("Could not dispatch command", e);
+            }
+            return;
+        }
+        Span span = spanFactory.createDispatchSpan(() -> "JGroupsConnector.processDispatchMessage", commandMessage)
```
Comment (smcvb) [inline jgroups/src/main/java/org/axonframework/extensions/jgroups/commandhandling/JGroupsConnector.java:353]: I think the method naming is a little off in this `CommandBusConnector`, as processing a `DispatchMessage` means receiving a message from the bus to provide to a handler.
So, I believe `SpanFactory#createChildHandlerSpan` is more aligned with the task at hand than `createDispatchSpan`.

**Review by smcvb — CHANGES_REQUESTED** [review]
All looks good, except for the invoked `SpanFactory` method. I believe we should use `createChildHandlerSpan` instead of `createDispatchSpan`.

**Review by smcvb — CHANGES_REQUESTED** [review]
All looks good, except for the invoked `SpanFactory` method. I believe we should use `createChildHandlerSpan` instead of `createDispatchSpan`.

**Comment by gklijs** [comment]
> All looks good, except for the invoked `SpanFactory` method. I believe we should use `createChildHandlerSpan` instead of `createDispatchSpan`.

Added, as well as mentioning the span factory in the javadoc.

**Review by smcvb — APPROVED** [review]
My concerns have been addressed, hence I'm approving this pull request.

**Pull request merged**